receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -19,10 +19,13 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io/ioutil"
+	"mime"
 	"net"
 	"net/http"
 	"sync"
 
+	"github.com/apache/thrift/lib/go/thrift"
```

Comment:
it's already imported as apacheThrift

Reply:
Thanks. BTW I reworked the implementation a bit in case you want to take another look.